internal/pkg_web/test3/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "pkg_web")
```

Comment:
"test3" isn't a descriptive name, can you make it "substitutions" or similar?

Reply:
Done ✅